src/Features/CSharp/Portable/Completion/KeywordRecommenders/UsingKeywordRecommender.cs

```diff
@@ -95,15 +95,7 @@ private static bool IsUsingDirectiveContext(CSharpSyntaxContext context, Cancell
             {
                 // root namespace
 
-                // a using can't come before externs
-                var nextToken = originalToken.GetNextToken(includeSkipped: true);
-                if (nextToken.Kind() == SyntaxKind.ExternKeyword ||
-                    ((CompilationUnitSyntax)context.SyntaxTree.GetRoot(cancellationToken)).Externs.Count > 0)
-                {
-                    return false;
-                }
-
-                return true;
+                return isValidContextAtTheRoot(context, originalToken, cancellationToken);
```

Comment:
IDE naming rule for local functions is to PascalCAse them :)  we think of them more as methods than as locals.